packages/ui/src/styles.css

```diff
@@ -348,12 +384,12 @@
     --canary-primary: 240 5.9% 10%;
     --canary-primary-background: 240 5.9% 10%;
     --canary-primary-foreground: 0 0% 98%;
-    --canary-primary-muted: var(--canary-grey-70);
+    --canary-primary-muted: var(--grey-70);
```

Comment:
RHS variable should be scoped/prefixed. 
this `--grey-70` will conflict with UICore `grey-70` and break both apps.
this is a blocker

Reply:
+1